### ISSUE TITLE
Coordination

### ISSUE THREAD

**Issue by harisont** [harisont/covid-19#2]
Please comment down here what sections of the article/images/... you are currently working on or planning to work on in the next 1-2 days, so we can complete this project in a relatively low amount of time without overlapping.

I will try to keep this list of tasks up to date, but also check the latest comments before starting to work:

- words.md:
  - [x] introduction (complete drat)
  - [x] the last few months (complete draft)
  - [x] the next few months (complete draft)
  - [x] the next few years (complete draft)
  - [x] the now (complete draft)

- other
  - [x] pics (complete draft)
  - [x] index.html (ongoing)
  - [x] sim/index.html (complete draft)
  - [x] thumbnail (complete draft)

**Comment by pietroppeter** [comment]
A note that was already in main ([Italian translation](https://github.com/ncase/covid-19/issues/3)) issue but might be useful also here for someone who wants to work on images. Many translations of the images can already be found here :

https://github.com/DP-3T/documents/tree/master/public_engagement/cartoon/it

**Comment by mendaomn** [comment]
Hey, I'm taking on `index.html` if that's ok

**Comment by simonefalcini** [comment]
I'm translating `sim/index.html` ok?

**Comment by simonefalcini** [comment]
How do you translate ICU? UTI?

**Comment by harisont** [comment]
> How do you translate ICU? UTI?

We don't think that acronym is actually used usually. So so far we just wrote "terapia intensiva", and I think according to the precise context it can become "letti di terapia intensiva", "posti in terapia intensiva", "reparto di terapia intensiva"... 

anyway for similar questions/suggestions you can take a look at #3

**Comment by simonefalcini** [comment]
I'm starting to translate the pics in alphabetical order, beginning with `pics/dp3t.png`

**Comment by harisont** [comment]
> I'm starting to translate the pics in alphabetical order, beginning with `pics/dp3t.png`

Great! Then if you stop, let us know here, otherwise we'll not touch the pictures.

**Comment by simonefalcini** [comment]
Ok, I'll write here when I stop, but I hope to finish all of them today :-)

**Comment by pietroppeter** [comment]
Thanks! Note that for dp3t there is already a translation available at the link mentioned above. It might not be ideal (or uniform with what we have, for example I see “contact tracing a prova di privacy”) but probably we can start with that and discuss later if we want to change it/uniform it with the rest.

**Edit**: ok you already saw that :)

**Comment by simonefalcini** [comment]
translation of `/pics` done.

**Comment by simonefalcini** [comment]
taking `the next few years`

**Comment by harisont** [comment]
We'll be done in no time! :)

**Comment by pietroppeter** [comment]
If we manage to have a first draft of all by the evening I can do a quick update of words.html and index.html so that for the proofreading part we can work on the pages version! I probably will be able to work this evening on reviewing overall translation (the parts that will be complete draft). I have at least one discussion point to raise (the FDR quote at the beginning I think deserves some additional explanation) but I expect to find more stuff to discuss. Great job everyone and especially @harisont for the overall coordination and leadership!

**Comment by harisont** [comment]
> If we manage to have a first draft of all by the evening I can do a quick update of words.html and index.html so that for the proofreading part we can work on the pages version! I probably will be able to work this evening on reviewing overall translation (the parts that will be complete draft). I have at least one discussion point to raise (the FDR quote at the beginning I think deserves some additional explanation) but I expect to find more stuff to discuss. Great job everyone and especially @harisont for the overall coordination and leadership!

It depends, if my evening you mean late night I will probably be done with my section at some point before sleep, but as for now Scenario 4+ is still at an early stage of translation cause I was giving a class during the past few hours. So if by evening you mean "now"... I'm not ready yet ;) 

But I'm pretty sure we can start reviewing tomorrow morning in any case. If I finish too late I can take care of updating the html file according to your directions myself, pandoc is always installed on my machine.

And thank **you** for your huge contribution!

**Comment by pietroppeter** [comment]
Yeah, I think I will call it a day and rest in the end... :P talk tomorrow!

**Comment by pietroppeter** [comment]
Ah, I checked and, as expected, there are some new changes in upstream repo. I have seen some commits related also to Scenario 4!

**Comment by harisont** [comment]
> Ah, I checked and, as expected, there are some new changes in upstream repo. I have seen some commits related also to Scenario 4!

Merging & solving conflicts right now ;)

**Comment by simonefalcini** [comment]
I finished translating `the next few years`

**Comment by harisont** [comment]
Finished `the next few months` too!

**Comment by harisont** [comment]
> If I finish too late I can take care of updating the html file according to your directions myself, pandoc is always installed on my machine.

@pietroppeter  done that, but [something is obviously not working](https://harisont.github.io/covid-19/). I probably did something stupid, or I have a different version of pandoc that requires different flags and stuff. In either case I don't have the energy to look into this right now, It's been intense ;)


Tomorrow is a busy day for me and I'll be available on and offm but will surely be able to do some reviewing at some point. Just don't wait for me to start, mine can be a second (or third) re-read (perché n occhi vedono meglio di 2)!

**Comment by pietroppeter** [comment]
I am pulling all changes and I will try to get the index.html working again. During the day I will not have time to review, should have more time in the evening.

**Comment by pietroppeter** [comment]
ok, pages now works okay. The output of your parsing was not protecting the html blocks, not sure why. The command I used is (recovered from my closed pull request #1 ) is:

    pandoc --from markdown_strict+footnotes words\words.md -o words\words.html

Now I will try to solve the conflicts on #9 and merge.

**Comment by pietroppeter** [comment]
ok, solved the conflict and also changed the translation notes. Now we can start reviewing and proofreading (there is still the thumbnail to complete for someone who wants to lend a hand!). How do we want to manage that? Everybody creates its review branch and we discuss over pull request and merge one by one resolving conflicts?

In the meantime I think we can open a pull request on upstream repo to add our translation (it will take time to be approved), do you want to do the honors @harisont ? :)

**Comment by pietroppeter** [comment]
Ah, another translation note: for consistency with index I changed the "simulazione giocabile" to "simulazione interattiva".

**Comment by pietroppeter** [comment]
Hey all! I [did something](f3ff43e1ec1aa7fa9648b7253cdd199374ad713f) that I think might be useful for this review phase. I added a `build` command (both for windows and linux, I tested only on windows since I am currently on windows).

What does this do?

* it runs the `pandoc` command on `words/words.md` to produce `words/words.html`
* it builds `index.html` from `index_head.html` + `words/words.html` + `index_tail.html`

To use this we should all make changes either to `words/words.md` or `index_head` or `index_tail` (*never directly on `index.html`*) and then run the `build` command to have them applied everywere. I did a test in first commit of my review branch and it seems to work fine. What do you think?

If you do not like or it does not work for you we can just remove the added files.

**Comment by harisont** [comment]
> How do we want to manage that? Everybody creates its review branch and we discuss over pull request and merge one by one resolving conflicts?

sounds a painful merge but I approve, it's probably the best way

> In the meantime I think we can open a pull request on upstream repo to add our translation (it will take time to be approved), do you want to do the honors @harisont ? :)

will do that now, but I'd rather make a draft pull request to mark as ready as soon as we have a decent mostly review version. You know, just in case it takes much less than we expect.

> I added a build command (both for windows and linux, I tested only on windows since I am currenlt on windows).

Great! I'm pretty sure I had used all the flags from you first PR yesterday, so now I'll test `build.sh` on Linux and if there are any problems adapt it. But most likely the problem is that we might have  different versions of pandoc. I'll let you know.

**Comment by pietroppeter** [comment]
my pandoc version is 1.19.2.1 (I use it on windows)

**Comment by harisont** [comment]
> my pandoc version is 1.19.2.1 (I use it on windows)

O well mine is 2.9.2... life on a rolling release hahah

**Comment by harisont** [comment]
One **important note** on reviewing: when you are about to make a substantial change, e.g. changing a very frequently used expression, do not forget we have #3 to discuss. Better not leave _all_ discussion to the merge step ;)

**Comment by harisont** [comment]
> > my pandoc version is 1.19.2.1 (I use it on windows)
> 
> O well mine is 2.9.2... life on a rolling release hahah

Anyway `build.sh` seems to be working for me. Not everything is converted exactly identically but it doesn't show in the rendered HTML, or at least so it looks to me.

**Comment by pietroppeter** [comment]
> > my pandoc version is 1.19.2.1 (I use it on windows)
> 
> O well mine is 2.9.2... life on a rolling release hahah

🤦 I did install pandoc the other day and the installer was a 2.9.2.1, but apparently I have in the path an older 1.19.2.1 coming from and Anaconda install...

Using the most recent pandoc I also see some differences and I agree they seem harmless. I will use the most recent pandoc from now on, so that we should not see differences between the two of us (after next time we commit a change using build command).

**Comment by simonefalcini** [comment]
i'm translating thumbnail.. 
Done!

**Comment by harisont** [comment]
My review is ready for review! Have "fun" on #14

**Comment by pietroppeter** [comment]
Thanks @harisont, the first thing I will do when I have a bit more time is review your review before doing another pass myself!

**Comment by pietroppeter** [comment]
Finally back! Now reviewing #14, then I will see what can I add to my review and try to list the open points we have. Hopefully I will be able to do all this today.

**Comment by pietroppeter** [comment]
...while I am doing the review #14, I realize I cannot request changes and approve commit by commit, right now it is approved but I will note down here when I am done (there will be at least one change request "Imparare a").

**Comment by pietroppeter** [comment]
Ok, I think went through all of #14 and added my comments. Thanks for the great work!

Every part on which there is no comment is approved on my part. I think I did not follow through the correct review "workflow" (I first did a "request changes" on a specific commit, an "approve" on another commit", then I started a review with lots of comments implying some "request changes" but finalized it with neither approval nor request changes"). Hopefully you will be able to sort this through otherwise let me know what can I do to help you with reviewing the review of the review...

Now, I will just check my notes to see if I had other changes to add to my PR (#12 ) and submit that for review, then I will start to summarizing some of the points still up for discussion.

**Comment by pietroppeter** [comment]
And #12 is also ready for review with minimal changes. I might be doing again an overall review but since @harisont did already such a great job I think our priority now should be to finalize this and make the translation official.

Thinking about outstanding issues that will be there after #12 and #14 are merged, I can note down:

* technical footnote #16 (I will try to do this later in a separate PR or still in #12 if it is not yet merged)
* circuit breaker: I do not think we decided anything about that. I did suggest [here](https://github.com/harisont/covid-19/issues/3#issuecomment-623542826) "spezza crescita" but did not receive feedback on that. We could keep English phrase but with respect to current translation I think we should still make some changes. (As above I will try to propose something for this later in a separate PR or still in #12 if it is not yet merged).
* check if there are commits from original repo to be merged (I gave a quick look and did not find anything relevant for the translation).
* feedback (and possibly official name) from @harisont on #11 (and relative changes)

After the above is all done we should:

* run the build script (making sure that changes made in index.html are also replicated in index_head.html, I am not sure this is the case for changes in #14!)
* and finalize the PR for adding translation in original repo!

Anything else that I am missing?

**Comment by pietroppeter** [comment]
I pushed a commit on #12 to close #16 (technical footnote). I will wait for review of #12 and merge of #14 before doing additional reviews.

**Comment by harisont** [comment]
@pietroppeter I addressed your comments and merged #14 . Unfortunately I am unable to go through all of your changes right away, I'll try (but can't promise) to do that tomorrow. In the meantime:

- thanks for translating #16 !
- circuit breaker: I can't think of a translation better than "spezza crescita" but I still understand the concept better in English
- I checked the original repo and updated our fork, nothing new to translate
- answered #11 :)
- ran build script but of course we'll need to do that again
- I think as soon as we merge your review and add credits we can finalize the pull request regardless minor changes we might want to make in the next few days!
- there was one thing I disagreed on in your review of my review: commas in floating point numbers. Are you sure that's the convention in technical texts in Italian? I do not think it is, but I don't read much technical stuff in Italian, after all. And if you are, could you please take care of this?

**Comment by pietroppeter** [comment]
Good, thanks! Later in the day I will probably be able to finalize most of the stuff that is left.

Regarding commas in decimal numbers I am positive that is the rule in Italian whether the text is technical* or not (commas to separate decimals, points to separate thousands). Word processors, spreadsheets and even DBs enforce that when the language is set to Italian (or [many others](https://it.wikipedia.org/wiki/Separatore_decimale)). I will take care of that.

*as example, here is a screenshot from a [relevant technical document](https://www.epicentro.iss.it/coronavirus/bollettino/Bollettino-sorveglianza-integrata-COVID-19_7-maggio-2020.pdf) (by ISS):
![image](https://user-images.githubusercontent.com/4997234/81906088-03522600-95c6-11ea-9819-ee253cd1c2a5.png)

**Comment by pietroppeter** [comment]
added a commit in #12 to close #11. I would wait for merge of #12 before the last points (commas for decimals and rerun build scripts). An eventual improvement for circuit breaker is lower priority now for me, but I guess I will eventually get to it.

**Comment by pietroppeter** [comment]
in #18 now the decimal separators are dealt with (along other small fixes). ready for review.
